test/metabase/api/card_test.clj

```diff
@@ -2873,19 +2873,20 @@
    (upload-example-csv! collection-id true))
   ([collection-id grant-permission?]
    (mt/with-current-user (mt/user->id :rasta)
-     (let [file              (upload-test/csv-file-with
+     (let [;; Make the file-name unique so the table names don't collide
+           csv-file-name     (str (random-uuid) ".csv")
```

Comment:
This was an issue when I updated `metabase-enterprise.advanced-permissions.common-test/upload-csv-test` to call this function multiple times in a `doseq` block